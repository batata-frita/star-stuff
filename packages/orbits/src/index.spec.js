const orbits = require('./index')

it('should generate a set of orbit with radius between 0 and 1', () => {
  expect(orbits(0.2)).toMatchSnapshot()
})
