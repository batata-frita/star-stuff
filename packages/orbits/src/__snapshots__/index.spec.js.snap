// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate a set of orbit with radius between 0 and 1 1`] = `
Array [
  0.05185344903845172,
  0.05156480477081028,
  0.05101996461409482,
  0.05071774745466213,
  0.05168777959730424,
  0.051840462231268564,
  0.05080423369853927,
  0.05004536217449483,
  0.050592486951039434,
  0.05157586534331203,
  0.05099274428978211,
  0.05089506307454342,
  0.05177828336577755,
  0.05001978467674268,
  0.050937979820448925,
  0.05105527580035142,
  0.050256952820366026,
  0.050867083543551996,
  0.05131836117552475,
]
`;
